cmd/entrypoint/runner_windows.go

```diff
@@ -32,11 +33,16 @@ import (
 
 // realRunner actually runs commands.
 type realRunner struct {
+	stdoutPath string
+	stderrPath string
 }
 
 var _ entrypoint.Runner = (*realRunner)(nil)
 
 func (rr *realRunner) Run(ctx context.Context, args ...string) error {
+	if rr.stdoutPath != "" || rr.stderrPath != "" {
```

Comment:
NIT: I guess we could have a runner_windows_test.go but not much to be tested really :D